.github/workflows/syncbranches.yml

```diff
@@ -0,0 +1,28 @@
+name: Merge fbsync to master
+
+on:
+  push:
+    branches:
+      - fbsync
```

Comment:
What happens if we sync one commit then sync another commit immediately after?

Example:

1. commit A is pushed
2. job A starts
3. commit B is pushed
4. job B starts (and finishes cloning the repo)
5. job A completes successfully
6. job B finishes cherry picking and pushes
7. job B fails because the push is declined as non-fast-forward

Right? Or am I missing something

Reply:
It is indeed might happen, although GitHub have a built in protection against some of that contingency (i.e. push event is not generated if commit B is committed less than 5 sec after commit A)
Also, this will fix itself automatically after "commit C" is landed sometimes later (whole logic of `syncbranches.py` is built around the notion of push events being unreliable)

In the long run, I'm thinking of switching this job to be executed periodically (say every 30 min) to avoid this and other such contingencies, but so far I want to try it as is.

Reply:
> P.S: I'm not 100% how github will treat those two histories as not fast forward, because "committer date" is getting updated anyway when commit is pushed to the repo, isn't it?

I didn't realize `cherry-pick` will always produce the same commit when copying. That's good to know